examples/demo/server/README.md

```diff
@@ -0,0 +1 @@
+# Demo Server
```

Comment:
Strange names "app" and "server" usually use "client" and "server", "frontend" and "backend", etc.

Please also fill the readme and explain what the server does, and how to run it.

Reply:
- Changed the names to _client_ and _server_ .

- I updated the main example README file to describe the client and the server as well as removed the Omnition load generators from the docker-compose file since it added a lot of data that was random and it was unclear what it represented.

- I updated the way it records measurements because it was broken.